src/org/ses/android/soap/FingerprintFindActivity.java

```diff
@@ -41,13 +45,16 @@
     private Button btnSearch;
 
     private AsyncTask<String, String, Participant> asyncTask;
+    private AsyncTask<String, String, String> asyncTask1;
```

Comment:
Can you give `asyncTask` and `asyncTask1` more descriptive names? Which Tasks are they generated from?